src/api/data/cache/cache.go

```diff
@@ -240,7 +240,11 @@ func (c *cachedCharts) Refresh() error {
 	for _, repo := range repos {
 		charts, err := repohelper.GetChartsFromRepoIndexFile(repo)
 		if err != nil {
-			return err
+			log.WithFields(log.Fields{
+				"repo": repo,
+				"error": err,
+			}).Error("Error on refresh charts from repo")
```

Comment:
the error message should start with a lowercase: "error on refresh..."

Reply:
Done